client/client.lua

```diff
@@ -344,48 +347,136 @@ local vehicledev_button = menu11:AddButton({
 local menu_dev_button = menu11:AddCheckbox({
     icon = '⚫',
     label = Lang:t("menu.hud_dev_mode"),
-    value = menu11,
+    value = nil,
     description = Lang:t("desc.hud_dev_mode_desc")
 })
 
-local deletelazer_button = menu11:AddCheckbox({
-    icon = '🔫',
-    label = Lang:t("menu.delete_laser"),
-    value = menu11,
-    description = Lang:t("desc.delete_laser_desc")
-})
 local noclip_button = menu11:AddCheckbox({
     icon = '🎥',
     label = Lang:t("menu.noclip"),
-    value = menu11,
+    value = nil,
     description = Lang:t("desc.noclip_desc")
 })
 
+local entity_view_button = menu11:AddButton({
```

Comment:
This variable is unused! What is the usage of it?

Reply:
This is used for setting the Initial menu under the `Developer Options Menu` which links to `menu15`. Which has all the settings for the new `Entity View Mode`.